### DISCUSSION TITLE
Free up device space by wiping

### DISCUSSION THREAD

**Discussion by UmmItKin** [ente-io/ente#1714] [Enhancements]
Hello, I recently transitioned entirely to ente and it's been an exceptional experience overall.

However, while exploring the Backup option, I noticed the option to free up device space. It got me thinking, does this process involve wiping data? If not, utilizing the wiping file method to free up space will be a safer alternative. It would ensure a more secure data removal process, which aligns with best for privacy.

**Comment by rcourtman** [comment]
The proxy belongs on the Proxmox host. Running it inside the LXC means it never gets access to `/run/pulse-sensor-proxy`, so systemd keeps restarting it.

Inside the container, run:
```
install-sensor-proxy.sh --uninstall
```

On the Proxmox host, install it against the Pulse container:
```
curl -fsSL https://raw.githubusercontent.com/rcourtman/Pulse/main/scripts/install-sensor-proxy.sh | bash -s -- --ctid <pulse-ctid> --pulse-server http://<pulse-ip>:7655
```

Restart the container so the bind mount is applied. Back on the host check the service:
```
systemctl status pulse-sensor-proxy
```

Inside the container confirm the socket bind is present:
```
ls -l /mnt/pulse-proxy
```

Once the host service is healthy and the mount is visible in the CT, the SSH log spam stops.
